typeset/mjnode.js

```diff
@@ -92,6 +92,7 @@ const convertMathML = async (log, mathMap/*: Map<string, {xml: string, fontSize:
   const convertedMathMLElements = new Map()
   let prevTime = Date.now()
   let numDone = 0
+  const convertedCss = new Map()
```

Comment:
Could it be just `Set` since we do not use values?

Reply:
I'll try using `Set`, thanks!